.code-samples.meilisearch.yaml

```diff
@@ -67,8 +67,10 @@ get_documents_1: |-
       }
   }
 add_or_replace_documents_1: |-
-  let path = Bundle.main.url(forResource: "movies", withExtension: "json")
+  let file = "movies.json"
```

Comment:
I think this one, is just to be a simple document addition 🤔 

<img width="950" alt="image" src="https://user-images.githubusercontent.com/4116980/163176812-51e0735b-66d7-4be0-9c8f-7451ca5fe4c5.png">